src/Powercord/coremods/plugin-embeds/components/PluginEmbed.jsx

```diff
@@ -37,7 +37,7 @@ const {
   subHead
 } = getModule([ 'titleRegion' ], false);
 
-module.exports = function ({ match }) {
+module.exports = function ({ match, props }) {
```

Comment:
seems a bit strange to take a nested props object, why don't we just take in an href and title prop?

Reply:
match is also a nested object. I don't personally see an issue with it.

Reply:
mmm, fair enough